pkg/kubectl/cmd/taint.go

```diff
@@ -366,23 +382,20 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 }
 
 // updateTaints updates taints of obj
```

Comment:
Please update the comment here. There's no point of having separate PR for that - we want to keep review load as low as possible.